1.9/installing/cloud/aws/basic.md

```diff
@@ -4,7 +4,7 @@ nav_title: AWS EC2 Basic
 menu_order: 0
 ---
 
-You can create a DC/OS cluster for Amazon Web Services (AWS) using the <a href="https://downloads.dcos.io/dcos/stable/1.9.1/aws.html" target="_blank">DC/OS templates for AWS CloudFormation</a>.
+You can create a DC/OS cluster for Amazon Web Services (AWS) using the <a href="https://downloads.dcos.io/dcos/stable/1.9.2/aws.html" target="_blank">DC/OS templates for AWS CloudFormation</a>.
```

Comment:
This is orthogonal to this PR....
I would need to change all links on the page for this to make sense... Let us rather do a cleanup PR which is focused on changing those links...
@judithpatudith @stbof